app/views/admin/setup.html.erb

```diff
@@ -0,0 +1,88 @@
+<h1>Create New Site</h1>
+
+<p class="is-lead">Welcome to your new <%= @community.name %> community!</p>
+
+<p>We are almost ready. We just need some more information to complete creation of the site.</p>
+
+<p>Don't worry. You will be able to change all these things in the site settings later.</p>
+
+<%= form_tag setup_save_url do %>
+
+<div class="widget"><div class="widget--body">

```

Comment:
**issue** (code-clarity): missing an indent level here after the opening `form_tag`. Also, shouldn't have two opening `<div>`s on the same line - separate lines with a new indent level for the second. Likewise the closing tags, and likewise the rest of this file.

Reply:
[![https://thecodinglove.com/content/041/cfkRtbl.webm](https://thecodinglove.com/content/041/cfkRtbl.webm)](https://thecodinglove.com/when-i-find-the-missing-div)